test/Unit/Porter/Net/Soap/SoapOptionsTest.php

```diff
@@ -58,6 +58,65 @@ public function testProxy()
         self::assertSame($password, $options->getProxyPassword());
     }
 
+    public function testEncoding()
+    {
+        self::assertSame($encoding = "iso-8859-1", (new SoapOptions)->setEncoding($encoding)->getEncoding());
```

Comment:
Since string interpolation is not being used, prefer expressing string literals using single quotes in such cases, for clearer expression of intent.